### DISCUSSION TITLE
(Cloud/Prem) Sustainability/GreenOps Maturity Model

### DISCUSSION THREAD

**Discussion by mkorbi** [cncf/tag-env-sustainability#38] [Project Proposal]
Hey there,

out of a discussion I had today with some end user customers, there was the question if we know a maturity model for cloud and/or on-prem regarding Environmental Sustainability/GreenOps etc. So we turned this discussion into a micro-workshop and came up with some simple five levels (for cloud):

1. Blind Optimization: non-evidence-based optimization e.g. shutdown of workload on off office hours/weekends etc. 
3. Manual Optimization based on CSP-provided dashboard: Analyzing of CO2e dashboards and planned actions to reduce those
4. Automated Performance & Utilization Optimization: Metrics based, automated optimization but not on Carbon
5. Carbon Aware Optimization: Like 4, but on carbon-relevant metrics
6. Carbon-Focused IT/Architecture: Having Sustainability as a leading principle within the enterprise architecture and IT strategy

**What would be this maturity model for?**

I think there would come a couple of things together, like from the landscape and the current possible options. The model should include per level a concrete list of actions so that organizations can plan for and target the next level.

A very rough idea, I couldn't find something on this at the GSF.

Thoughts & feedback?

**Comment by leonardpahlke** [comment]
I like the idea, this could be a project in close collaboration with the GSF (maybe even GSF takes the lead here?) since this is not specific to Cloud Native but still very relevant for us at the end.

**Comment by Marksbutcher** [comment]
If you're looking to create a digital sustainability maturity model, you need to consider more than the technical side. A lot of the work in embedding digital sustainability spans strategy, culture, governance and standards alongside the more typical operational and design efficiencies. 

It's important to understand organisational requirements and processes today for reporting on sustainability and how it dovetails into wider ESG programmes. Also important to know their organisational structure, reporting cadence and current capability

**Comment by Marksbutcher** [comment]
The FinOps Foundation sustainability working group has created a series of standard FinOps related remediations for controlling cloud resource usage with respect to GreenOps and are currently creating a methodology for assessing level of sustainability awareness (chose to call it awareness rather than maturity as maturity implies you have a benchmark against which to measure someone)

**Comment by mkorbi** [comment]
Covered here https://github.com/cncf/tag-env-sustainability/issues/50
